src/packaging/packages.targets

```diff
@@ -1,37 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <UsingTask TaskName="GetBuildUid" TaskFactory="CodeTaskFactory" AssemblyName="Microsoft.Build.Tasks.v12.0">
-        <ParameterGroup>
-            <BuildUid ParameterType="System.String" Output="true" />
-        </ParameterGroup>
-        <Task>
-            <Code>
-                BuildUid = System.Guid.NewGuid().ToString().Replace("-", "").Substring(0, 8);
```

Comment:
There was an env variable `NUGET_PACKAGES` which didn't work very well earlier. Can we try setting this at the  start of repo build.cmd to be repo local `$CORERT/packages`, so we can clean up? In Unix setting `HOME` makes the cache repo local.

@anurse, the problem is we are building `corert` repo to produce uniquely named ILC packages which we restore and publish. This is to avoid conflicts with other build processes. Can we specify an override for the package cache which all dotnet commands (like restore/publish) and nuget would strictly adhere to?